CONTRIBUTING.md

```diff
@@ -187,13 +189,90 @@ pip install -e ./
 
 It's going to install Addons in editable mode without compiling anything.
 You can modify source files and changes will be seen at the next Python 
-interpreter startup.
+interpreter startup. This command needs to be executed only once. 
+Now, anywhere on your system, if you do `import tensorflow_addons`, it's 
+going to import the code in this git repository.
+
+To undo this operation, for example, you want to later on 
+install TensorFlow Addons from PyPI, the release version, do:
+
+```
+pip uninstall tensorflow-addons
+```
+
+If TensorFlow Addons is installed in editable mode, you can then just run your tests by 
+running Pytest. For example:
+```bash
+pip install -r tools/install_deps/pytest.txt
+python -m pytest tensorflow_addons/rnn/cell_test.py
+# or even
+python -m pytest tensorflow_addons/rnn/
+# or even 
+python -m pytest tensorflow_addons/
+# or even if pytest is in the PATH
+pytest tensorflow_addons/
+```
+
+Pytest has many cool options to help you make great tests:
+
+```bash
+# Use multiprocessing to run the tests, 3 workers
+pytest -n 3 tensorflow_addons/
+pytest -n auto tensorflow_addons/
+
+# Open the debugger to inspect variables and execute code when 
+# an exception is raised.
+pytest --pdb tensorflow_addons/ 
+
+# or if you prefer the Ipython debugger
+pytest --pdb --pdbcls=IPython.terminal.debugger:TerminalPdb --capture no tensorflow_addons/
+
+# by defaults print() aren't displayed with pytest
+# if you like to debug with prints (you might get 
+# the outpout scrambled
```

Comment:
```suggestion
# the output scrambled)
```

Curious as to what kind of scrambling would happen here? Different tests printing overtop of each other or out of order for a single test?